data/templates/accel-ppp/config_chap_secrets_radius.j2

```diff
@@ -5,7 +5,7 @@ chap-secrets={{ chap_secrets_file }}
 [radius]
 verbose=1
 {%   for server, options in authentication.radius.server.items() if not options.disable is defined %}
-server={{ server }},{{ options.key }},auth-port={{ options.port }},acct-port={{ options.acct_port }},req-limit=0,fail-time={{ options.fail_time }}
+server={{ server }},{{ options.key }},auth-port={{ options.port }},{% if options.disable_accounting is defined %}acct-port=0{% else %}acct-port={{ options.acct_port }}{% endif %},req-limit=0,fail-time={{ options.fail_time }}

```

Comment:
Why not adjust the accounting port to 0 as with the lagacy implementation?

Reply:
I'm afraid I don't understand how it works. And what do I need to do.

Reply:
I'd extend the common `get_accel_dict()` functionality which can and should be used by any Accel-PPP service in the same way you did for `src/conf_mode/vpn_l2tp.py`. Thus it's coded one time on value retrieval and no hacks in templates are required. Chat me up on Slack if you need more infos.

Reply:
I have updated the PR and am not changing the template for sstp "config_chap_secrets_radius.j2"